"npm run once" runs zero

Currently, it gives me an error and blows up.

```
$ npm run once

> activity-streams@0.0.1 once /Users/pdehaan/dev/github/activity-streams
> npm run bundle && firefox

...

> activity-streams@0.0.1 bundle:html /Users/pdehaan/dev/github/activity-streams
> node ./bin/generate-html.js > data/content/activity-streams.html

sh: firefox: command not found

npm ERR! Darwin 15.3.0
npm ERR! argv "/Users/pdehaan/.nvm/versions/node/v5.6.0/bin/node" "/Users/pdehaan/.nvm/versions/node/v5.6.0/bin/npm" "run" "once"
npm ERR! node v5.6.0
npm ERR! npm  v3.6.0
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! activity-streams@0.0.1 once: `npm run bundle && firefox`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the activity-streams@0.0.1 once script 'npm run bundle && firefox'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the activity-streams package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run bundle && firefox
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs activity-streams
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls activity-streams
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/pdehaan/dev/github/activity-streams/npm-debug.log
```

The suspect line, is of course:

> ### sh: firefox: command not found

via [./package.json:81](https://github.com/mozilla/activity-streams/blob/master/package.json#L81):

```js
    "once": "npm run bundle && firefox",
    "firefox": "jpm run -b beta",
```

Not sure if that should be `npm run firefox` instead of just `firefox`.
Initial investigation points to this recent change: https://github.com/mozilla/activity-streams/commit/b3511001cf5aa2b8b8d567471b37c314c43664cf#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L84

```diff
   # once: Build/serve the assets and run the add-on
-  once: =>bundle && npm-run-all --parallel start:embedly firefox
+  once: =>bundle && firefox
```

--- a/package.json
+++ b/package.json
@@ -77,7 +77,7 @@
     "yamscripts": "^0.0.2"
   },
   "scripts": {
-    "once": "npm run bundle && firefox",
+    "once": "npm run bundle && npm run firefox",
     "clean": "rm -rf data/content/*",
     "changelog": "conventional-changelog -i CHANGELOG.md -s",
     "bundle": "npm-run-all bundle:*",

--- a/package.json
+++ b/package.json
@@ -99,7 +99,7 @@
     "test:jpm": "jpm test -b beta",
     "test:karma": "karma start --browsers FirefoxNightly",
     "tdd": "karma start --no-single-run",
-    "package": "jpm xpi && mv @activity-streams-$npm_package_version.xpi dist/activity-streams-$npm_package_version.xpi",
+    "package": "npm run bundle && jpm xpi && mv @activity-streams-$npm_package_version.xpi dist/activity-streams-$npm_package_version.xpi",
     "travis": "npm-run-all travis:*",
     "travis:eslint": "npm run test:lint",
     "travis:jpm": "jpm test -b $JPM_FIREFOX_BINARY -v",

--- a/yamscripts.yml
+++ b/yamscripts.yml
@@ -6,7 +6,7 @@
 scripts:
 
 # once: Build/serve the assets and run the add-on
-  once: =>bundle && firefox
+  once: =>bundle && =>firefox
 
   clean: rm -rf data/content/*
 

--- a/yamscripts.yml
+++ b/yamscripts.yml
@@ -46,7 +46,7 @@ scripts:
   tdd: karma start --no-single-run
 
 # package: Build add-on
-  package: jpm xpi && mv @activity-streams-$npm_package_version.xpi dist/activity-streams-$npm_package_version.xpi
+  package: =>bundle && jpm xpi && mv @activity-streams-$npm_package_version.xpi dist/activity-streams-$npm_package_version.xpi
 
   # This is for ci
   travis:

